classes/checkout/CheckoutPersonalInformationStep.php

```diff
@@ -33,6 +33,11 @@ class CheckoutPersonalInformationStepCore extends AbstractCheckoutStep
 
     private $show_login_form = false;
 
+    /**
+     * @var bool
+     */
+    public $logged_in;
```

Comment:
Not sure about the visibility. Let me know if we should be worried about automatically setting these properties to `public` by phpstan.

Reply:
For not changing the behavior, the default scope is public :)